assets/training/aoai/proxy_components/src/common/azure_openai_client_manager.py

```diff
@@ -47,7 +47,7 @@ def _get_workspace_subscription_id_resource_group(self) -> str:
         """Get current subscription id."""
         uri = os.environ.get(AzureOpenAIClientManager.MLFLOW_TRACKING_URI, None)
         if uri is None:
-            return None
+            return None, None
```

Comment:
Why this is needed? Return type is just `str` for this function

Reply:
actually return type is tuple[str,str], fixed the return type